composer.json

```diff
@@ -11,9 +11,9 @@
     "nikic/php-parser": "^4.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^8.5 || ^9.0",
+    "phpunit/phpunit": "^9.6.18",
```

Comment:
`^9.0` is fine here.

Reply:
I don't remembered exactly what, but there was an error when run lower version of phpunit v9

Reply:
Could we approve this and move on? Or this needs further investigation? As related packages are waiting for an update

Reply:
> I don't remembered exactly what, but there was an error when run lower version of phpunit v9

Indeed, works fine with `^9.3` though, I edited your commit accordingly.